pkgs/games/katago/default.nix

```diff
@@ -37,22 +40,28 @@ assert !enableCuda -> (
 assert enableTcmalloc -> (
   gperftools != null);
 
+assert enableContrib -> (
```

Comment:
This is also not required.

Reply:
Thanks, I've adjusted this and the documentation.